ranger/container/fsobject.py

```diff
@@ -82,6 +82,7 @@ class FileSystemObject(  # pylint: disable=too-many-instance-attributes,too-many
     document = False
     image = False
     media = False
+    application = False
```

Comment:
What do you have in mind for this? It's for application-specific formats that don't fit the other top-level types, usually binary data.